File: src/BukkitPE/event/block/BlockFormEvent.php
<?php
namespace BukkitPE\event\block;

use BukkitPE\block\Block;
use BukkitPE\event\Cancellable;

class BlockFormEvent extends BlockGrowEvent implements Cancellable{
	public static $handlerList = null;

	public function __construct(Block $block, Block $newState){
		parent::__construct($block, $newState);
	}

}

File: src/main/java/net/BukkitPE/event/block/BlockFormEvent.java
package net.BukkitPE.event.block;

import net.BukkitPE.block.Block;
import net.BukkitPE.event.Cancellable;
import net.BukkitPE.event.HandlerList;

/**
 * BukkitPE Project
 */
public class BlockFormEvent extends BlockGrowEvent implements Cancellable {

    private static final HandlerList handlers = new HandlerList();

    public BlockFormEvent(Block block, Block newState) {
        super(block, newState);
    }

    public static HandlerList getHandlers() {
        return handlers;
    }

}
